front/main/app/components/wikia-ui-components/text-field.js

```
@@ -71,6 +73,19 @@ export default Ember.Component.extend({
 			if (typeof onFocusHandler === 'function') {
 				onFocusHandler(event);
 			}
+		},
+
+		/**
+		 * @param {String} value
+		 * @param {Event} event
+		 * @returns {void}
+		 */
+		onKeyUp(value, event) {
+			const onKeyUpHandler = this.get('onKeyUpHandler');
+
+			if (typeof onKeyUpHandler === 'function') {
```

Maybe we could change this logic to only check if handler was defined, in other case let it fail. That way silent failure, when wrong handler type provided, won't happen.